core/spec.md

```diff
@@ -1233,11 +1234,9 @@ Where:
   to ensure that the value matches the entity's current `epoch` value and if it
   differs then an error MUST be generated.
 - If the entity's unique identifier is present in the object, then it MUST
-  match its corresponding key value. And for Versions, if `RESOURCEid` is
-  present then it MUST match the Resource's `SINGULARid` (`rID`) specified
-  in the URL.
+  match its corresponding `KEY` value.
```

Comment:
I don't quite follow this change - hoping you'll explain it in the meeting.

Reply:
it's just saying that:
```
  "v1": {
    "versionid": "v2"
  }
```
is invalid, "versionid" should have been "v1". Is it the phrase "entity's unique identifier" that's confusing?

Reply:
It's what was *removed* that's confusing me - in terms of why we don't want that any more.

Reply:
Ah, I didn't want implementations to have to jump through hoops to check other fields when technically they don't matter. The only one that's critical is "epoch" (if present). I kept "id" as something to check just because if it doesn't match the KEY then your json itself is wrong and you might be deleting the wrong thing, which is different than forcing the impl to check with the backend/DB for verification. If you think checking the rID is still needed (if present) I can revert that part.

Reply:
I don't know enough about the rID part to have a meaningful opinion - I'd suggest just talking it through in the meeting later, and hopefully it'll all be fine. (I don't have a concrete objection.)

Reply:
So I don't forget... one other reason I excluded rID is that in order to even know what the rID attribute's name is (e.g. schemaid), the impl would need to go to the model/DB and ask for it. When everything was just "id" that kind of look-up wasn't necessary. I can't imagine that's a big deal, but it feel into the category of "how much work do we want the impl to do just to 'double check' the user on a 'delete'?"  Yes, let's discuss on the call - not much of an agenda anyway :-)